.dockerignore

```diff
@@ -1,3 +1,4 @@
 Makefile
 .env
 .git
+kube
```

Comment:
I think we can get rid of this `.dockerignore` file now that it is split into modules. Although the Dockerfile might be a bit longer it'll be also safer because we need to explicitly add files instead of removing (we might forget). 

For example, if I'm quickly testing the pipeline and add a temporary file with secrets or something else its easy to forget about ignoring it before building and pushing the image to Dockerhub.

Would love to hear how does that sounds for you!